package.json

```diff
@@ -94,8 +95,5 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ],
-  "resolutions": {
-    "fsevents": ">=2.0.0"
-  }
+  ]
```

Comment:
Is removing this config related to this feature?

Reply:
it's my mistake, just for local testing, it has been restored.